src/Components/CreateImageWizardV2/steps/Packages/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { Text, Form, Title } from '@patternfly/react-core';
+
+const PackagesStep = () => {
+  return (
+    <Form>
+      <Title headingLevel="h2">Additional packages</Title>
```

Comment:
same

Reply:
Answered here https://github.com/osbuild/image-builder-frontend/pull/1640#discussion_r1494424651, the "Additional packages" heading is correct in this case :)